src/validations/valuesetValidation.ts

```diff
@@ -15,6 +15,20 @@ const GetValueSetErrors = async (
       valuesetsArray.map(async (valueSet) => {
         const oid = getOidFromCqlValueSet(valueSet, model);
         const locator = getLocatorFromCqlValueSet(valueSet);
+        if (model === "QICore") {
+          const valuesetUrlWithoutOid = valueSet?.url?.split("ValueSet/");
+          if (
+            valuesetUrlWithoutOid &&
+            valuesetUrlWithoutOid[0] !== "'http://cts.nlm.nih.gov/fhir/"
+          ) {
+            const invalidValuesetUrlError =
+              processValueSetErrorForElmTranslationError(
+                `"${valueSet.url} is not a valid URL"`,
+                locator
+              );
+            valuesetsErrorArray.push(invalidValuesetUrlError);
```

Comment:
if a URL is not valid, should we skip the call to `terminologyServiceApi.getValueSet()` ?

Reply:
It looks like we could first perform checks to make sure the ValueSet is valid and if valid, we can proceed to get OID , Locator and make a call to API

Reply:
No this would be seperate, because getValuset is concerned only with oid check suppose consider a situation where both url and oid are not valid ones in that case we need to provide 2 errors validation error for oid and also for the url.